src/scripts.js

```diff
@@ -44,162 +46,164 @@ searchBar.addEventListener('keyup', searchRecipes);
     recipes.forEach(recipe => {
-        let titleClass = 'recipe-title'
-        if (recipe.name.length > 17) titleClass = 'recipe-title long-title';
         let heartClass = "heart-icon";
         if(!recipe.saved) heartClass = "heart-icon hidden";
         recipeSection.innerHTML += `
         <article class="recipe-card pointer" data-parent="${recipe.id}" tabindex="0">
-            <img class="recipe-img" src="${recipe.image}" data-parent="${recipe.id}" alt="Picture of ${recipe.name}">
-            <img class="${heartClass}" data-parent="${recipe.id}" src="./images/heart.png" alt="This recipe is in my recipes!">
-            <h2 class="${titleClass}" data-parent="${recipe.id}">${recipe.name}</h2>
+                <img class="recipe-img" src="${recipe.image}" data-parent="${recipe.id}" alt="Picture of ${recipe.name}">
+                <img class="${heartClass}" data-parent="${recipe.id}" src="./images/heart.png" alt="This recipe is in my recipes!">
+                <h2 class="recipe-title" data-parent="${recipe.id}">${recipe.name}</h2>
         </article>`;
     });
 }
 
 function createRecipeModal(event) {
-   if(event.target.className !== "all-recipes" || event.key === 'Enter') {
-    toggleHidden(modalSection);
-    body.classList.add('no-scroll')
-    let recipeID = +(event.target.dataset.parent);
-    modalRecipe = recipeRepo.recipes.find(recipe => recipe.id === recipeID);
-    modalSection.innerHTML = `
-    <div class="recipe-popup">
-        <img role="button" aria-label="Close Recipe Button" class="close-icon pointer" id="closeIcon" src="./images/close-icon.png" tabindex="0">
-        <h2>${modalRecipe.name}</h2>
-        <div class="print-container">
-          <div class="image-ingredients">
-          <img class="modal-img" src="${modalRecipe.image}" alt="${modalRecipe.name} image">
-          <ul class="oregano-font">
-              <h3>Ingredients:</h3>
-              ${createList(modalRecipe.listIngredients(ingredients))}
-          </ul>
-          </div>
-          <ol class="oregano-font">
-            <h3>Directions:</h3>
-            ${createList(modalRecipe.getInstructions())}
-          </ol>
-        </div>
-        <h4 class="oregano-font"><i>TOTAL COST $${modalRecipe.listCost(ingredients)}</i></h4>
-        <div class="button-container">
-          <button class="modal-button pointer" id="saveBtn" tabindex="0">${updateButtonText()}</button>
-          <button class="modal-button pointer" id="printBtn">Print Me!</button>
-        </div>
-    </div>`;
-    document.querySelector('#closeIcon').focus()
-    document.getElementById('saveBtn').addEventListener('click', toggleSaveRecipe);
-    document.getElementById('printBtn').addEventListener('click', () => window.print());
-    document.getElementById('closeIcon').addEventListener('keypress', (event) => {
-      event.key === "Enter" ? collapseRecipe(event) : null
-    });
-  }
+    if(event.target.className !== "all-recipes" || event.key === 'Enter') {
```

Comment:
Looks like some weird formatting stuff? I'll try to make a pass tomorrow to refactor and clean up a bit